packages/react-components/react-theme/stories/Theme/typography/ThemeTypography.stories.tsx

```diff
@@ -1,51 +1,66 @@
 import * as React from 'react';
-import { teamsLightTheme, typographyStyles } from '@fluentui/react-components';
-import type { TypographyStyle } from '@fluentui/react-components';
-
-const theme = teamsLightTheme;
-
-export const TypographyStyles = () => {
-  // var(--tokenName) => tokenName
-  function formatTypographyStyleValue(typographyStyleValue: TypographyStyle) {
-    return (
-      <div>
-        {Object.values(typographyStyleValue).map(value => (
-          <div key={value}>{value.replace(/var\(--(.+)\)/, '$1')}</div>
-        ))}
-      </div>
-    );
-  }
-
-  // caption1Strong => Caption 1 Strong
-  function formatTypographyStyleName(typographyStyleName: string) {
-    return typographyStyleName.replace(/([A-Z\d])/g, ' $1').replace(/^(.)/, firstChar => firstChar.toUpperCase());
-  }
+import { makeStyles, Subtitle2Stronger, Text, TypographyStyles } from '@fluentui/react-components';
+import { typographyStyles } from '@fluentui/react-components';
+
+const useStyles = makeStyles({
+  container: {
+    rowGap: '24px',
+    columnGap: '48px',
+    display: 'grid',
+    gridTemplateColumns: 'auto auto 1fr',
+    alignItems: 'start',
+  },
+
+  ...typographyStyles,
+});
+
+export const Typography = () => {
+  const styles = useStyles();
+  const tokenOrder: (keyof TypographyStyles)[] = [
```

Comment:
this whole custom generation should be in separate function outside of this component. storybook  rendering is problematic on its own. this will add overhead to our docs if left as is, on every re-render all of these regexes and operations will need to be re-created which is expensive.

So it should look something like 

```
const tokens = generateTokens();

export const Typography = () => {
  const styles = useStyles();
  // ... resst
}
```

Reply:
That makes sense. Updated as well.